### PR TITLE
fix(plus): add breadcrumbs for Collections/Notifications/Settings

### PR DESCRIPTION
**Pull request by caugner**
## Summary

Adds breadcrumbs for Collections/Notifications/Settings in MDN Plus.

### Problem

MDN Plus did not have breadcrumbs (except for the "Documentation" section).

### Solution

Make it easy to add breadcrumbs to non-Doc pages.

---

### Before

image

### After

image

---

## How did you test this change?

Checked out these pages locally:

- http://localhost:3000/en-US/plus/collections
- http://localhost:3000/en-US/plus/notifications
- http://localhost:3000/en-US/plus/settings

### CODE REVIEW

File: client/src/plus/index.tsx [hunk 1]
```diff
@@ -18,6 +21,10 @@ export function Plus({ pageTitle, ...props }: { pageTitle?: string }) {
     document.title = pageTitle || MDN_PLUS_TITLE;
   }, [pageTitle]);
 
+  const { locale = "en-US" } = useParams();
+  const { pathname } = useLocation();
+  const userData = useUserData();
```
Comment (Guyzeroth) [inline client/src/plus/index.tsx:26]: Not used.

**Review by Guyzeroth — CHANGES_REQUESTED** [review]

**Comment by Guyzeroth** [comment]
1 change then LGTM. Thanks for picking that up 👍

File: client/src/plus/index.tsx (hunk 1 above)
Comment (caugner) [reply client/src/plus/index.tsx:26]:
```suggestion
```

File: client/src/plus/index.tsx [hunk 2]
```diff
@@ -9,6 +9,9 @@ import Notifications from "./notifications";
 import { MDN_PLUS_TITLE } from "../constants";
 import { Settings } from "../settings";
 import PlusDocs from "./plus-docs";
+import { useUserData } from "../user-context";
```
Comment (caugner) [inline client/src/plus/index.tsx:12]:
```suggestion
```

**Comment by Guyzeroth** [comment]
Coolio. Thanks! :)

**Pull request merged**